src/thread/thread_entry.cpp

```diff
@@ -328,6 +370,46 @@ namespace cubthread
     pthread_mutex_unlock (&th_entry_lock);
   }
 
+  void
+  entry::end_resource_tracks (void)
+  {
+    if (!ENABLE_TRACKERS)
+      {
+	// all trackers are activated by this flag
+	return;
+      }
+    m_alloc_tracker.clear_all ();
+    m_pgbuf_tracker.clear_all ();
+    m_csect_tracker.clear_all ();
```

Comment:
Shouldn't m_csect_tracker be stopped as well after clear?

Reply:
fixed by resetting m_start_count